### ISSUE TITLE
How to get averaged loss in multi-gpu training ?

### ISSUE THREAD

**Issue by shuuchen** [pytorch/pytorch#47491]
Hi,

I am using multi-gpu training, following the tutorial:
https://pytorch.org/docs/stable/notes/ddp.html

I am trying to construct the curves of training and validation losses for visulization. But it seems I can only access the loss of one gpu.

I know that the losses of multi-gpu will be averaged before back propagation. So how to get the averaged loss ? 

Thank you !

**Comment by SsnL** [comment]
You can perform a reduce over the ranks. Github is not for Q&A. Please use the forum.
